src/Microsoft.TemplateEngine.Abstractions/PrecedenceDefinition.cs

```diff
@@ -34,7 +34,7 @@ public enum PrecedenceDefinition
     /// <summary>
     /// Parameter might become disabled - value would not be needed nor used in such case.
     /// </summary>
-    ConditionalyDisabled,
+    ConditionallyDisabled,
```

Comment:
we could avoid the breaking change by adding the corrected enum value and hiding the existing one

```csharp
[EditorBrowsable(EditorBrowsableState.Never)]
ConditionalyDisabled
```
and
```csharp
ConditionallyDisabled = ConditionalyDisabled
```

Reply:
or mark ConditionalyDisabled as obsolete

Reply:
or both :)